netket/exact.py

```diff
@@ -143,7 +143,7 @@ def full_ed(operator, compute_eigenvectors=False):
         return EdResult(w, None)
 
 
-def steady_state(lindblad, sparse=False, method="ed", rho0=None, **kwargs):
+def steady_state(lindblad, sparse=None, method="ed", rho0=None, **kwargs):
```

Comment:
Maybe make the options here keyword-only as well, while we're at it?
```suggestion
def steady_state(lindblad, *, sparse=None, method="ed", rho0=None, **kwargs):
```